networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -139,13 +139,61 @@ include::modules/nw-multus-tap-object.adoc[leveloffset=+2]
 
 * For more information about enabling an SELinux boolean on a node, see xref:../../nodes/nodes/nodes-nodes-managing.adoc#nodes-nodes-working-setting-booleans_nodes-nodes-managing[Setting SELinux booleans]
 
+// Set of includes pertains to OVN-Kubernetes additional network
 include::modules/configuring-ovnk-additional-networks.adoc[leveloffset=+2]
+
+[id="{context}_configuration-additional-network-types-supported-platforms"]
+==== Supported platforms for OVN-Kubernetes additional network
+
+You can use an OVN-Kubernetes additional network with the following supported platforms:
+
+- bare-metal
+- {ibmpowerProductName}
+- {ibmzProductName}
+- {linuxoneProductName}
+- VMware vSphere
+- {rh-openstack}
+
+// Workaround lack of xref in modules
+[id="{context}_configuration-additional-network-types-prerequisites"]
+==== Prerequisites for configuring OVN-Kubernetes additional network
+
+- Install NMstate Operator. For more information, see xref:../../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#"k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator]
+
+[id="{context}_configuration-additional-network-interface"]
+==== Configuration for an OVN-Kubernetes additional network mapping
+
+You must map the additional network to the OVN bridge to use it as an OVN-Kubernetes additional network. You can create an `NodeNetworkConfigurationPolicy` object, part of the `nmstate.io/v1` API, to declaratively create the mapping.
+
+The network-admin would provision the following NNCP (nmstate policy object) to add a mapping from the network with name `localnet-network` to the `ovsbr1`:
```

Comment:
I feel this example should be as concise as possible, but also correct. 

For that, I would represent the scenario where the `br-ex` OVS bridge is shared. Thus, please use br-ex here, and in the example below.